### PR TITLE
[KYUUBI #1052] Implement api: /sessions (POST)

### PR DESCRIPTION
**Pull request by yanghua**
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: kyuubi-server/pom.xml [hunk 1]
```diff
@@ -86,6 +86,28 @@
...
+            <groupId>org.glassfish.jersey.test-framework</groupId>
+            <artifactId>jersey-test-framework-core</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.glassfish.jersey.test-framework.providers</groupId>
+            <artifactId>jersey-test-framework-provider-jetty</artifactId>
+            <scope>test</scope>
+            <exclusions>
```
Comment (turboFei) [inline kyuubi-server/pom.xml:99]: these exclusions has been declared in parent pom.
Should we remove them here?

File: kyuubi-server/src/main/java/org/apache/kyuubi/server/api/v1/dto/SessionOpenedCount.java [hunk 2]
```diff
@@ -33,4 +34,21 @@ public int getOpenSessionCount() {
...
+    if (!(o instanceof SessionOpenedCount)) {
+      return false;
+    }
+    SessionOpenedCount that = (SessionOpenedCount) o;
+    return getOpenSessionCount() == that.getOpenSessionCount();
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(getOpenSessionCount());
```
Comment (turboFei) [inline kyuubi-server/src/main/java/org/apache/kyuubi/server/api/v1/dto/SessionOpenedCount.java:52]: Invoke openSessionCount directly? 
return Objects.hash(openSessionCount);

**Review by turboFei — APPROVED** [review]

File: kyuubi-server/src/main/java/org/apache/kyuubi/server/api/v1/dto/SessionOpenedCount.java (hunk 2 above)
Comment (yanghua) [reply kyuubi-server/src/main/java/org/apache/kyuubi/server/api/v1/dto/SessionOpenedCount.java:52]: OK

**Comment by yanghua** [comment]
@turboFei Thanks for your review. Have addressed your concerns.

**Comment by turboFei** [comment]
thanks, merged to master

**Pull request closed without merging**